unit_test/conftest.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--runslow", action="store_true", default=False, help="run slow tests"
```

Comment:
I'm not sure we should have this as `--runslow`. Rather something like `--with-docker` or something? It's not per se about slow, but about not "polluting" the unit test environment with docker images (for which I meant the `--runslow` example from the pytest docs might be a good demonstration of how to achieve that)?